portal/src/graphql/adminapi/RolesScreen.module.css

```diff
@@ -1,12 +1,19 @@
 .content {
-  @apply mt-11;
+  flex: 1 1 0;
```

Comment:
To fix the scrollbar problem, it seems you need overflow: hidden; here

Reply:
I think should just add `overflow: auto;` here

Reply:
No, because the scrollbar should appear in content of CommandBarContainer.